shop/src/Components/Navbar/index.jsx

```diff
@@ -0,0 +1,92 @@
+import Logo from "../../Assets/Icons/Logo";
+import StoreIcon from "../../Assets/Icons/StoreIcon";
+import LoginIcon from "../../Assets/Icons/LoginIcon";
+import HumbergerIcon from "../../Assets/Icons/HumbergerIcon";
+import MobileMenu from "../MobileMenu";
+import "./Navbar.css";
+import { BrowserRouter as Router, NavLink } from "react-router-dom";
+import useIsOnMobile from "../../hooks/useIsOnMobile";
+import { useState } from "react";
+import { useEffect } from "react";
+const USER_MAIN_MENU = [
+  { path: "/men", link: "men" },
+  { path: "/kids", link: "kids" },
+  { path: "/women", link: "women" },
+];
+const ICONS = [
+  { path: "/store", component: <StoreIcon /> },
+  { path: "/login", component: <LoginIcon /> },
+];
+
+const Navbar = () => {
+  const onMobile = useIsOnMobile();
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
```

Comment:
why do we need a state `isMenuOpen`? can we just use `onMobile` ?

Reply:
I think that, onMobile is base on screen width. isMenuOpen is base on click to open or close mobileMenu.